docs/_docs/03-ug-mutable-dataset.md

```diff
@@ -90,4 +97,139 @@ hs.refreshIndex("empIndex", "incremental")
 TODO
 
 ### Hybrid Scan
-TODO
+
+Hybrid Scan enables to utilize existing index data along with newly appended source files or
+deleted existing files, without explicit refresh operation. For an index with appended source data files,
+HybridScan changes the query plan to shuffle new data on-the-fly and merge it with index records.
+For an index with deleted source data files, Hyperspace also modifies the plan to exclude the rows from deleted files
```

Comment:
```suggestion
For an index with deleted source files, Hyperspace also modifies the plan to exclude the rows from deleted files
```

Reply:
I am keeping it consistent this as "source files" as you first used it in line 101. (Can you check other places?)